app/Actions/CreateOrEditConfig.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Actions;
+
+use Illuminate\Support\Facades\File;
+
+class CreateOrEditConfig
+{
+    public function __invoke()
+    {
+        $configPath = config('home_dir') . '/.lambo';
+        $configFilePath = "{$configPath}/config.json";
+
+        if (File::exists($configFilePath)) {
+            app('console')->info("Opening existing config file: {$configFilePath}");
+            $this->editConfig($configFilePath);
+            return 0;
```

Comment:
Why return `0`?

Reply:
Just being a good command line citizen. I guess it's not totally required.